File: web-development/level-1/helsinki-part-1-2/Python/part1/week02/ex39.py
#ex 39 week02 part1
#Printing stars

#methods
def printStars(amount):
    print('*' * amount)

def printSquare(side_size):
    for i in range(side_size):
        print('***')

def printRect(width, height):
    for i in range(height):
        print('*' * width)

def printTriangle(size):
    for i in range(size):
        printStars(i)


#Main
printStars(9)
print('')
printSquare(3)
print('')
printRect(10, 5)
print('')
printTriangle(10)

File: programming/helsinki-moocs/java/ex39.java
import java.util.Scanner;

public class ex39 {

  public static void main(String[] args) {
    print_rect(10,3);
  } // entry point

  public static void printStars(int number) {
    for (int i = 0; i < number; i++) {
      System.out.print('*');
    }
    System.out.println("");
  }

  public static void print_square(int side_size) {
    for (int i = 0; i < side_size;  i++) {
      printStars(side_size);
    }
  }

  public static void print_rect(int width, int height) {
    for (int i = 0; i < height; i++) {
      printStars(width);
    }
  }
}
